tailwind.config.ts

```diff
@@ -245,8 +240,8 @@ module.exports = {
         }
       };
       addUtilities(newUtilities);
-    },
-    ({ addUtilities, theme, variants }) => {
+    }),
+    plugin(({ addUtilities, theme, variants }) => {
```

Comment:
I'm not sure if these plugins are still needed. Tailwind 3 might provide these already.